config/tcs_config.toml

```diff
@@ -14,7 +14,7 @@
 
 [KvStorage]
 StoragePath = "config/Kv_Shared_tmp"
-StorageSize = "1 TB"
+StorageSize = "1 GB"
```

Comment:
1 GByte is very small. The space for the file is not allocated until used, so 1 TByte allows room to grow.

Reply:
1 GB is only for local test, which won't be included in the final commit.